edge/poll_async.go

```diff
@@ -58,7 +59,9 @@ func (service *PollService) startStatusPollLoopAsync() {
 
 	log.Debug().Msg("starting Portainer async polling client")
 
-	var snapshotFlag, commandFlag, coalescingFlag bool
+	snapshotFlag := true
+	commandFlag := true
```

Comment:
Why are you setting these flags to `true`? Please keep the original declaration.